ajax/libs/geomicons-open/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "geomicons-open",
+  "filename": "geomicons.min.js",
+  "version": "2.0.0",
+  "description": "Open Source Icons for the Web",
+  "homepage": "http://geomicons.com/",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/jxnblk/geomicons-open.git"
+  },
+  "author": "Brent Jackson",
+  "keywords": [
+    "icons",
+    "css",
+    "svg"
+  ],
+  "npmName": "geomicons-open",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*.!(json)"
```

Comment:
Let's use this pattern `"**/!(*json|*.gz)"`
Because the current pattern still can get `*.gz` file which to be excluded.
Thanks